src/test/java/subway/RestAssuredTest.java

```diff
@@ -16,8 +21,21 @@ public class RestAssuredTest {
     @Test
     void accessGoogle() {
         // TODO: 구글 페이지 요청 구현
-        ExtractableResponse<Response> response = null;
+        String targetURL = "https://google.com";
+
+        RestAssured.baseURI = targetURL;
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Accept", "*/*");
+        headers.put("Accept-Encoding", "gzip, deflate, br, zstd");
+        headers.put("Accept-Language", "ko-KR,ko;q=0.9,en-US;q=0.8,en;q=0.7");
 
+        ExtractableResponse<Response> response = given()
+                                                .headers(headers)
+                                                .when()
+                                                .get()
+                                                .then()
+                                                .statusCode(200)
+                                                .extract();
```

Comment:
```suggestion
        ExtractableResponse<Response> response = given()
                .headers(headers)
                .when()
                .get()
                .then()
                .statusCode(HttpStatus.OK.value())
                .extract();
```

- 줄맞춤을 해주면 좋을 것 같아요!
- statusCode는 하드코딩으로 200을 입력하는것 보다는 열거타입을 활용해봐도 좋을 것 같아요!

Reply:
남겨주신 것을 이제 확인했습니다..! 이번에 같이 반영하겠습니다. 감사합니다!